.github/workflows/build_hub.yml

```diff
@@ -91,6 +76,35 @@ jobs:
           # This path is from the root of the repo as needed by the plugin
           files: prime-router/build/test-results/testIntegration/**/*.xml
           check_name: "Integration Test Results"
+          
+  linting:
+    name: Check Linting
+    needs: pre_job
+    if: ${{ needs.pre_job.outputs.has_router_change == 'true' }}
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: prime-router
+
+    steps:
+      - name: "Check out changes"
+        uses: actions/checkout@v2
+
+      - name: Run Linting Check
+        run: bash ./build.sh -- gradle ktlintCheck
```

Comment:
I have no such task in my branch.  Where is this coming from?

Reply:
it's setup [here](https://github.com/CDCgov/prime-reportstream/blob/master/prime-router/build.gradle.kts#L33) and the including the plugin [automatically adds the tasks](https://github.com/JLLeitschuh/ktlint-gradle)